frontend/src/page/training.rs

```diff
@@ -621,18 +641,19 @@ pub fn view_table<Ms: 'static>(
                             }
                         ],
                         td![&t.load()],
-                        td![&t.set_volume()],
-                        td![common::value_or_dash(t.avg_rpe())],
+                        IF![has_set_volume_data => td![common::value_or_dash(t.set_volume())]],
```

Comment:
I have the feeling that set volume should always have a value and never be represented by a dash. A dash would mean that no value can be given. But as each training session consists of sets, it should always be possible to determine a value. If there are no hard sets for a training session, 0 should be shown. Consequently, the set volume column should be always displayed.

Reply:
Right bro, if ya ain't do no hard sets at all, ya should get a zero right in ya face :wink:

I agree. What do you think about renaming the `Set volume` column to reflect that fact? Something like `Hard set volume`. The downside is that this is wrapped into three lines on small screens.

Reply:
The term `Set volume` is typically used for the number of hard sets. See for example [this article](https://fitguide.blog/training/volume/#Setvolume). I think the better solution would be to add a description for this term (and also other terms and abbreviations). This could be a help page (something that would be nice to have in the long term) or a short description that is shown when hovering/tapping the name. The latter is already used for the muscle group tags.

Reply:
Thanks for the link - that's a nice summary article! I kept "Set volume" as is then. I agree that a help page or descriptions would be nice in the future (at the same time, I don't feel like investing a lot of time in this now when I read up on the topic).